packages/apps/plugins/plugin-navtree/src/NavTreePlugin.tsx

```diff
@@ -2,27 +2,48 @@
 // Copyright 2023 DXOS.org
 //
 
+import { MagnifyingGlass } from '@phosphor-icons/react';
 import React from 'react';
 
 import {
+  type GraphBuilderProvides,
+  resolvePlugin,
   type MetadataRecordsProvides,
   type PluginDefinition,
   type SurfaceProvides,
   type TranslationsProvides,
+  type Plugin,
+  parseIntentPlugin,
+  LayoutAction,
+  type GraphProvides,
```

Comment:
i wish lint would consistently sort this. types first? consts, function, classes, etc. ;)